lib/plugins/aws/package/lib/iam-role-lambda-execution-template.json

```diff
@@ -21,12 +21,7 @@
           "Statement": [
             {
               "Effect": "Allow",
-              "Action": ["logs:CreateLogStream"],
-              "Resource": []
-            },
-            {
-              "Effect": "Allow",
-              "Action": ["logs:PutLogEvents"],
+              "Action": ["logs:CreateLogGroup", "logs:CreateLogStream", "logs:PutLogEvents"],
```

Comment:
Interesingly in #6241, it was assumed that resource definition for `logs:CreateLogGroup` and `logs:PutLogEvents` cannot be merged -> https://github.com/serverless/serverless/pull/6212#issuecomment-499643350
(cc @rdsedmundo @pmuens )

I take, that after all they can be merged (so for `logs:PutLogEvents`, `:*` trailing can be stripped)

Reply:
Yeah, I thought that it could be a problem, but if you tested and it worked without the :* we should be good.

Reply:
I didn't :)

@pmuens when testing, did you cover the case of creating new function, invoking it, and confirming that logs for it are visible in CloudWatch?

Reply:
Hi everyone, this is my first PR, your feedback and support are really appreciated. 

@medikoo I have not tested this specific scenario yet. I will test it and update in this thread.

Reply:
I have tested it using this simple repo: https://github.com/azizur/serverless-pull-6692 a completely new project and it worked. I have previously tested with my existing project already, so there was no need to test again.

I have tested with additional modification too:

1) remove `*:*` from https://github.com/serverless/serverless/blob/15dce40ce0284cf04e4c92754e85118a81cfec47/lib/plugins/aws/customResources/index.js#L136

It worked since it generated policy resource refrences like:
```
"arn:aws:logs:{AWS_REGION}:{AWS_ACCOUNT}:log-group:/aws/lambda/{stage}-{function-a}*:*"
```

Then I tried with additional modifications with `mergeIamTemplates.js`

1) remove `:*` from https://github.com/serverless/serverless/blob/15dce40ce0284cf04e4c92754e85118a81cfec47/lib/plugins/aws/package/lib/mergeIamTemplates.js#L104

2) remove `*:*` from https://github.com/serverless/serverless/blob/15dce40ce0284cf04e4c92754e85118a81cfec47/lib/plugins/aws/package/lib/mergeIamTemplates.js#L113

It worked since it generated policy resource refrence like:
```
"arn:aws:logs:{AWS_REGION}:{AWS_ACCOUNT}:log-group:/aws/lambda/{stage}-{function-a}"
```

I think we can go with the additional modifications above and it will work. Looking forward to feedback from others.

Reply:
Apologies for change to my previous message. I did not fully understand all the test scenarios. I have now checked the following policies and I can confirm that:
```
"PolicyDocument": {
              "Version": "2012-10-17",
              "Statement": [
                {
                  "Effect": "Allow",
                  "Action": [
                    "logs:CreateLogGroup",
                    "logs:CreateLogStream",
                    "logs:PutLogEvents"
                  ],
                  "Resource": [
                    {
                      "Fn::Sub": "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/{function-a}*:*"
                    }
                  ]
                }
              ]
            }
```
Works: 
-  :white_check_mark: creates log groups
-  :white_check_mark: creates log streams 
-  :white_check_mark: Log stream as logs entries. <<< This was missed from my above testing.

When I modified https://github.com/serverless/serverless/blob/15dce40ce0284cf04e4c92754e85118a81cfec47/lib/plugins/aws/package/lib/mergeIamTemplates.js Line 104 and Line 113 in 15dce40.

Works: 
-  :white_check_mark: creates log groups
-  :white_check_mark: creates log streams 

Did not work:
-  :x: Log stream as logs entries. <<< This was missed from my above testing.

Reply:
Great thanks @azizur for thorough check, we really appreciate it.

Let us know on final outcome.